Yosemite/Yosemite/Actions/AnnouncementsAction.swift

```diff
@@ -0,0 +1,13 @@
+import Storage
+
+/// AnnouncementsAction: Defines all of the Actions supported by the AnnouncementsStore.
+///
+public enum AnnouncementsAction: Action {
+
+    public typealias IsCached = Bool
+
+    /// Synchronizes the latest Announcements.
+    /// If information was already fetched, the list of features will be retrieved along with the boolean indicating that the data was already fetched
+    ///
+    case synchronizeFeatures(onCompletion: ([WooCommerceFeature], IsCached) -> Void)
```

Comment:
Some comments here:

1. Why synching only features and not the full announcement? I think we could use some announcement information(like the app version) to make sure we don't show the wrong data

2. Why `IsCached` is needed? I think `synchronizeFeatures` should always sync/request the latest announcement from the network.

3. We may also be missing an action to load them without downloading them. This can be in another PR tho.